lib/main.coffee

```diff
@@ -188,6 +188,32 @@ module.exports =
         return p
     return execPath
 
+  getProjectPath: (lintedFile) ->
+    projectPaths = atom.project.getPaths()
+    projectNames = projectPaths
+      .map (projectPath) -> (
+        splittedPath = projectPath.split('/')
+        return splittedPath[splittedPath.length - 1]
+      )
+
+    projectName = projectNames
+      .find((projectName) -> lintedFile.indexOf(projectName) > -1)
+
+    return projectPaths[projectNames.indexOf(projectName)]
+
+
+  getExistingConfigFiles: (configFiles, projectPath) ->
+    return configFiles
+      .split(/[ ,]+/)
```

Comment:
The config option mentions this is only the file name, but this seems to turn it into an array of _potential_ file names?